build.sbt

```diff
@@ -936,4 +956,4 @@ lazy val root = (project in file("."))
   )
 
 addCommandAlias("assemblySamples", ";flinkManagementSample/assembly;standaloneSample/assembly;demo/assembly;generic/assembly")
-addCommandAlias("assemblyEngines", ";flinkProcessManager/assembly;engineStandalone/assembly")
+addCommandAlias("assemblyEngines", ";flinkProcessManager/assembly;flinkPeriodicProcessManager/assembly;engineStandalone/assembly")
```

Comment:
I think it's not needed ATM, as periodic manager currently can be used only via custom extension (we don't expose any ProcessManagerProvider via ServiceLoader)